### PR TITLE
feat: add getting started resources to homepage, project pages, all projects page and bridge page

### PR DESCRIPTION
**Pull request by blushi**
## Description

ref: regen-network/regen-registry#1372

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] added new items content to `./deskStructure.js`
- [ ] go through ["Deploying to production" instructions](https://github.com/regen-network/regen-sanity/blob/main/README.md#deploying-to-production) after this PR (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] manually tested (if applicable)

### CODE REVIEW

File: schemas/documents/shared/gettingStartedResourcesCard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+export default {
```
Comment (blushi) [inline schemas/documents/shared/gettingStartedResourcesCard.js:1]: set this as a document so content editors can reuse a same card within different "Resources for getting started" sections if needed in the future
this also allows us to share the same content for the "bridging" card that is displayed both on its own on the bridge page and within other cards on "Resources for getting started" sections

File: schemas/documents/shared/gettingStartedResourcesSection.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export default {
```
Comment (blushi) [inline schemas/documents/shared/gettingStartedResourcesSection.js:1]: set this as a document so a same section with the same cards can be reused on multiple pages
this is currently the case on the homepage, all projects page and individual project pages

**Review by mhagel — APPROVED** [review]

**Review by wgwz — APPROVED** [review]
approving w trust :-) i don't fully understand the details but it does seem pretty straightforward

**Comment by blushi** [comment]
merging this before the front-end PR https://github.com/regen-network/regen-web/pull/1613 since it is non api breaking

**Pull request merged**